quizzes/quiz3.md

```diff
@@ -0,0 +1,221 @@
+    }
+}
+```
+
+Valid but not logically correct. The conditions that age is smaller, equal, or larger than 6 are mutually exclusive conditions, meaning that they can not be true at the same time. For conditions that are mutually exclusive we have a construct which we use, it is called if, else if, and else. So the correct version will be:
+
+```js
+if ( age < 6 ) {
+    // do something
+} else if ( age === 6 ) {
+    // do something else
+} else {
+    // do something completely different
+}
+```
+
+#### B.
+
+```js
+<script src="main.js">
+  console.log("hi");
+</script>
+```
+
+Valid but not doing what we want. It seems like we both want to execute main.js file and want to execute the contents of the `<script>` tag. But when the src attribute is set the contents of the `<script>` tag is ignored. To solve this, we can create two separate `<script>` tags.
+
+```js
+<script src="file.js"></script>
+<script>
+  console.log("hi");
+</script>
+```
+
+## THEORETICAL PART:
+
+#### 1. Do you still remember what we learnt in this week?
+
+**HTML attribute**
+
+Attributes provide additional information about HTML elements. We use them in the HTML documents. They are typed inside the opening tag of an HTML element and their content is put inside double quotes. Examples are:
+
+**script** element has **src** attribute.
+```html
+<script src="..."></script>
+```
+
+**button** element has **onclick** attribute.
+```html
+<button onclick="..." >click me</button>
+```
+
+**input** element has **id** attribute.
+```html
+<input id="..."/>
+```
+
+**input** element has **value** attribute.
+```html
+<input value="..." />
+```
+
+**input** element has **type** attribute.
+```html
+<input type="..."/>
+```
+
+**input** element has **placeholder** attribute.
+```html
+<input placeholder="..."/>
+```
+
+**body** element has **background-color** attribute.
+```html
+<body background-color="...">
+</body>
+```
+
+HTML is **not case sensitive**, you can use any small or capital letters for the attribute names. But stick to all lower case letters when you type them in your HTML.
+
+___
+
+**HTML DOM**
+
+In JavaScript code, we want to manipulate the web page contents, meaning we want to add new elements to the page, we want to change the style of elements, we want to react to events when the user does something on the page.
+
+For this purpose, a web browser provides programmers **a language** that can be used inside JavaScript code. We call this language DOM. Document Object Model. With DOM, JavaScript can create a dynamic HTML page.
+
+This language is made up of **only objects**, specifically HTML objects. They are called **HTML objects** because each object in this language represents an HTML element in the HTML document. **Why** are they **objects** but not something else is because JavaScript uses objects natively just like it uses strings and numbers. **Important to remember**, in HTML we talk about elements, on JavaScript we call these elements objects.
```

Comment:
```suggestion
This "language" is made up of **only objects**, specifically HTML objects. They are called **HTML objects** because each object in this language represents an HTML element in the HTML document. **Why** are they **objects** but not something else is because JavaScript uses objects natively just like it uses strings and numbers. **Important to remember**, in HTML we talk about elements, on JavaScript we call these elements objects.
```

Reply:
```suggestion
This language is made up of *objects only**, specifically HTML objects. They are called **HTML objects** because each object in this language represents an HTML element in the HTML document. **Why** are they **objects** but not something else? It is because JavaScript uses objects natively just like it uses strings and numbers. **Important to remember**, in HTML we talk about elements, in JavaScript we call these elements objects.
```